### PR TITLE
Add a persistent filter to our model

### PR DESCRIPTION
**Pull request by feeds**
Create a new Filter that is saved in `http://127.0.0.1:8000/matching/A/filter/create`

or call the rest api to filter the existing participants with get parameters `http://127.0.0.1:8000/matching/api/A/info/list` (see  OPTIONs-button)

### CODE REVIEW

**Comment by feeds** [comment]
@bjrne can you explain why the tests are failing? I have no idea.. :grin:

**Comment by feeds** [comment]
@maltezacharias This is what I should you ... what else do you need?  :)

File: backend/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -18,6 +18,18 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "X-Generator: Poedit 2.3.1\n"
 
+msgid "is exactly"
+msgstr ""
```
Comment (bjrne) [inline backend/locale/de/LC_MESSAGES/django.po]: I guess because of this :D

**Review by bjrne — DISMISSED** [review]

**Comment by feeds** [comment]
Use #74 instead

**Pull request closed without merging**